### DISCUSSION TITLE
Question: retrocompatibility with spacing from Bootstrap 4

### DISCUSSION THREAD

**Discussion by tomasts248** [twbs/bootstrap#34320] [Q&A] [answered]
I noticed on my site that migrating from B4 to 5 moved everything a bit, playing and comparing the code I can revert that by using the following code (at the end), but I wonder can the new default be made so that resembles the exact position of elements without using retro code? and how wise is to use this snippet in terms of good practices?

Edit : most likely I should adjust the values of the variables to match the old behavior instead? I apologize for the ignorance.

This is the code:
```
  .row > * {
    padding-right: 15px;
padding-left: 15px;
  }
  .container-fluid, .container-lg, .container-md, .container-sm, .container-xl {
    padding-right: 15px;
    padding-left: 15px;
    margin-right: auto;
    margin-left: auto;
  }

.row {
  margin-right: -15px;
  margin-left: -15px;
}

hr {
  margin-top: 1rem;
  margin-bottom: 1rem;
  border: 0;
  border-top-color: currentcolor;
  border-top-style: none;
  border-top-width: 0px;
  border-top: 1px solid rgba(0,0,0,.1);
  opacity:1;
  background-color: unset;
}
```

**Comment by mdo** [comment, marked as answer]
That snippet is fine. The main reason we migrated our spacing is alignment with our own utilities—so `.gx-3` is the same as `.mx-3`, and the default grid spacing can be restored or modified with ease. Previously this wasn't possible with Bootstrap 4, so changing back to that will remove that capability unless you modify the utilities as well.

**Comment by tomasts248** [comment]
Thank you for explaining :) 

> The main reason we migrated our spacing is alignment with our own utilities—so .gx-3 is the same as .mx-3

Understood more consistent behavior.

> unless you modify the utilities as well.

Question if I want to do that can I just change the variable in css to $grid-gutter-width: 30px; 
I have the impression I'm missing important parts.
